pkgs/development/interpreters/python/hooks/default.nix

```diff
@@ -1,9 +1,9 @@
-self: super: with self;
+self: dontUse: with self;
```

Comment:
I want it to be explicit that it should not be used, It could be changed to `_dontUse` if one wanted to quieten `deadnix`.

Reply:
BTW, why exactly will this never be useful or usable at any time in the future?

I can see why it isn't useful right now, but it seemed non-obvious to me that it would never be useful.  If that's true there probably ought to be a comment explaining it.

Reply:
If super had to be used then it won't be possible to deduplicate and inherits would have to be done manually
```nix
      keep = self: {
        inherit (self)
          condaInstallHook
          condaUnpackHook
          eggBuildHook
          eggInstallHook
          eggUnpackHook
          flitBuildHook
          pipBuildHook
          pipInstallHook
          pytestCheckHook
          pythonCatchConflictsHook
          pythonImportsCheckHook
          pythonNamespacesHook
          pythonOutputDistHook
          pythonRecompileBytecodeHook
          pythonRelaxDepsHook
          pythonRemoveBinBytecodeHook
          pythonRemoveTestsDirHook
          setuptoolsBuildHook
          setuptoolsCheckHook
          setuptoolsRustBuildHook
          sphinxHook
          unittestCheckHook
          venvShellHook
          wheelUnpackHook
          wrapPython
        ;
      };
```


calling twice with self probably won't work when the package actually has a difference between self and super, I don't know when that would be the case here

Reply:
The best thing to solve this issue would be https://github.com/NixOS/nixpkgs/pull/228139 and to [deprecate python3.pkgs ](https://github.com/NixOS/nixpkgs/issues/211340)

Reply:
Of course that is assuming I debugged correctly and there's no other issues.

Reply:
> to [deprecate python3.pkgs ](https://github.com/NixOS/nixpkgs/issues/211340)
> but it's not urgent

:+1: